tests/e2e/install-codeflare-sdk.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+cd ..
+
+# Install Poetry and configure virtualenvs
+pip install poetry
+poetry config virtualenvs.create false
+
+cd codeflare-sdk
+
+# Lock dependencies and install them
+poetry lock --no-update
```

Comment:
I am wondering why we run `poetry lock --no-update` if the `poetry.lock` file should already exist?

Reply:
When we run `poetry lock --no-update` we are resolving the dependencies from the project.toml file, which then creates the `poetry.lock` file. You're right, the poetry.lock file already exists, but if any changes are made to the project.toml file we would need to update our poetry.lock file too. The mentioned command ensures project.toml and poetry.lock are in sync.